common/lib/user.js

```diff
@@ -1,5 +1,6 @@
-function User({ name, roles = [], locations = [] } = {}) {
-  this.userName = name
+function User({ userName, fullName, roles = [], locations = [] } = {}) {
+  this.userName = userName
```

Comment:
Mentioned earlier but maybe this should be `username` as it's actually only one word.

Reply:
In fact, do we even need this anymore? It's not being used anywhere in the app. Could just simply things and remove it?